### ISSUE TITLE
Rethink display and selection of registries

### ISSUE THREAD

**Issue by nichtich** [gbv/cocoda#694]
With growing number of mapping sources the display and selection of mapping registries becomes inconvenient. So far we have:

- coli-conc registry (main, read/write)
- Local registry (also writing)
- Wikidata registry (only supports mappings from Wikidata, also writing)
- Konsys registry (only supports mappings from Konstanz Systematik, no writing)

If this list grows we should rethink how to best display and select registries.

**Comment by nichtich** [comment]
The "data sources" view in settings should do, but it is fairly hidden. Maybe add a link to the MappingBrowser settings to "manage data sources"?

**Comment by guitarster** [comment]
There is also the option to de-/select registries with a click on one of the buttons which are in line with the "Mappings" headline: C, L, W. Same at the section "Suggestions": CRe, CO and CC. For active users of Cocoda, I think, this option is enough to handle this issue at the moment. If a new registry and / or suggestion tab will be added sometime, there could be an update notice to the active users in time, so they can decide to de-/select this new tab as well. But yes, for new users too much tabs could be irritating. I think one possible way could be to select only coli-conc registry, Local registry and Wikidata registry per default. All other registries should be deselected.

**Comment by stefandesu** [comment]
I agree with David that de-/selecting registries with the buttons should be enough to manage registries, even with 1-2 more configured registries. At some point, I guess the Cocoda instances admin should decide which registries aren't relevant and remove them from the configuration.

I also don't mind the idea of deselecting certain registries by default.

Adding an additional "manage data sources" button to MappingBrowser settings is also a good idea, even if it's simply an alias to go to the right tab in Settings.
